.gitignore

```diff
@@ -81,6 +81,7 @@ dlldata.c
 *.pidb
 *.svclog
 *.scc
+*.dll
```

Comment:
We do have ~18 small .dlls committed in into the repo for testing and other purposes. This will hide them...

Reply:
Yeah I'll remove that. I didn't want to accidentally commit msquic...